src/components/AdobeTab.tsx

```diff
@@ -42,9 +42,11 @@ export default class AdobeTab extends React.Component<AdobeTabProps, AdobeTabSta
             let name = library.basic_info.name;
             let numberOfPatrons = parseInt(library.basic_info.number_of_patrons);
               return (
-                <li key={name} className={hasStyles ? "adobe-data-li" : ""}>
-                  <span>{name}: {numberOfPatrons} </span>
-                  <span>({getPercentage(numberOfPatrons, this.props.data.map(l => l.basic_info.number_of_patrons), true)})</span>
+                <li key={name} className={hasStyles ? "adobe-li" : ""}>
+                  <section className={hasStyles ? "header-bar" : ""}>
+                    <span>{name}: {numberOfPatrons} </span>
```

Comment:
For someone not familiar with this UI, you'll see "library_name: 6". It makes more sense to include that it's 6 patrons. Not part of the PR but just noticed that when switching between tabs and, specifically, when looking at the yearly data tab, it says "2019: 7" but doesn't explain much more.

Reply:
That...is a very good point.

Reply:
(For the yearly data tab, it's not the number of patrons; it's the number of new libraries.  I guess it's REALLY confusing!  Will fix.)

Reply:
Right. In that case maybe it should be "libraried added" or something.